projects/adapter/src/dbt/adapters/fal_experimental/support/athena.py

```diff
@@ -0,0 +1,82 @@
+from typing import Any
+import six
+from dbt.adapters.base.relation import BaseRelation
+from dbt.contracts.connection import AdapterResponse
+import sqlalchemy
+import pandas as pd
+from dbt.adapters.base import BaseAdapter
+from urllib.parse import quote_plus
+
+
+def create_engine(adapter: BaseAdapter) -> Any:
+    creds = adapter.config.credentials._db_creds
```

Comment:
Looks like athena credentials are `adapter.config.credentials`.

Just judging from this error:

```
16:22:06  Unhandled error while executing /tmp/tmpkwcled1j/target/run/source_test/models/model_a.py
'AthenaCredentials' object has no attribute '_db_creds'
Error: 16:22:06.287093 [error] [Thread-1 (]: Unhandled error while executing /tmp/tmpkwcled1j/target/run/source_test/models/model_a.py
'AthenaCredentials' object has no attribute '_db_creds'
16:22:06  1 of 1 ERROR creating python table model ***_custom.model_a ...... [ERROR in 16.62s]
Error: 16:22:06.289416 [error] [Thread-1 (]: 1 of 1 ERROR creating python table model ***_custom.model_a ...... [ERROR in 16.62s]
16:22:06  
16:22:06  Finished running 1 table model in 0 hours 0 minutes and 24.81 seconds (24.81s).
16:22:06  
16:22:06  Completed with 1 error and 0 warnings:
16:22:06  
16:22:06  'AthenaCredentials' object has no attribute '_db_creds'
Error: 16:22:06.302038 [error] [MainThread]: 'AthenaCredentials' object has no attribute '_db_creds'
16:22:06  
```